Scripts/DDIBuilderHelpers.eol

```diff
@@ -968,19 +968,97 @@ operation m!MultiLangString BuildLangString(lang : String, content : String) : m
   The 'value' of the TaggedValue
  @comments
   Builds a TaggedValue, adds it to the context ModelElement and returns it 
-  <br> Note that a MultiLangString does not have an explicit 'key' or 'value' property; 
-  <br> the key is set as the lang of a LangString contained in the MultiLangString, the 'value' being the content 
   <br> The built TaggedValue is initialised with an ID using GetID() 
+  <br><b>Attention: Method updated to comply to SACM meta model:</b>
+  <br> Each TaggedValue consists of a MultiLangString <key> containing LangStrings.
+  <br> The key value is now in the "content" attribute of the LangString element <value>.	
+  <br> It is further now possible to add multiple keys for one TaggedValue. 
+  <br> See functions below.	
+  <br> Each TaggedValue further has a MultiLangString <content> to store the value
+  <br> as a LangString. Here, the value is also located in the "content" attribute 
+  <br> of the LangString element <value>. 
+  <br> It is now also possible to add multiple values to one or more keys within
+  <br> a single TaggedValue element. See functions below.
 */
 operation m!ModelElement BuildTaggedValue(key : String, value : String) : m!TaggedValue {
 	var result : new m!TaggedValue;
 	result.key = new m!MultiLangString;
-	result.key.BuildLangString(key, value);
+	result.key.BuildLangString("en", key );
+	result.content = new m!MultiLangString;
+	result.content.BuildLangString("en",value);
```

Comment:
Try to maintain the style consistent i.e. leave a space after the comma

Reply:
Added spaces accordingly